README.md

```diff
@@ -1,24 +1,5 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
+Todey we larn ruby
+Anonimous Rubist Club (ARC)
```

Comment:
You've done some mistakes in this text, correct it to "Today we are going to learn ruby", "Anonymous Rubyist Club(ARC)" 🍾